.github/workflows/dapr.yml

```diff
@@ -49,32 +49,54 @@ jobs:
             target_os: linux
             target_arch: amd64
             job_name: "Linux/amd64"
+            sidecar_flavor: "all"
```

Comment:
Given the different variables on each entry, we have to manually generate the permutations we need. We need to rethink how we do this because the Windows builds created this situation.